playwright_fill_timesheet.spec.js

```diff
@@ -0,0 +1,64 @@
+const { test, expect } = require('@playwright/test');
+
+test('test', async ({ page }) => {
+
+  test.setTimeout(0);
+
+  const delay = ms => new Promise(resolve => setTimeout(resolve, ms))
+  await delay(0);
+
+  const username = process.env.TIMESHEET_USERNAME
+  const password = process.env.TIMESHEET_PASSWORD
+  const url = process.env.TIMESHEET_URL
+
+  await page.goto(url);
+  await page.locator('input[name="pf\\.username"]').click();
+  await page.locator('input[name="pf\\.username"]').fill(username);
+  await page.locator('input[name="pf\\.pass"]').click();
+  await page.locator('input[name="pf\\.pass"]').fill(password);
+  await page.locator('text=Submit').click();
+  await expect(page).toHaveURL(url);
+
```

Comment:
```suggestion
await page.close({ runBeforeUnload: true })
```

Just showing how to, if we no longer need the tab open